Frontend/ui-customer/src/Helpers/AuthHelper.purs

```diff
@@ -0,0 +1,44 @@
+module Helpers.AuthHelpers where
+
+import Prelude
+import Services.API (TriggerSignatureOTPResp(..), User(..))
+import Data.Either (Either(..))
+import Presto.Core.Types.API (ErrorResponse)
+import Data.Maybe (Maybe(..), fromMaybe, isNothing)
+import Engineering.Helpers.BackTrack (liftFlowBT)
+import Storage (setValueToLocalStore, KeyStore(..), getValueToLocalStore)
+import Effect.Uncurried (runEffectFn2)
+import Control.Monad.Except.Trans (lift)
+import Presto.Core.Types.Language.Flow (setLogField)
+import Foreign.Class (encode)
+import Helpers.Utils (getMobileNumber)
+import Common.Types.App (SignatureAuthData(..), EventPayload(..))
+import Types.App (FlowBT)
+import Data.Function.Uncurried (runFn3)
+import JBridge (emitJOSEvent)
+import Accessor (_authData)
+import Data.Lens ((^.))
+
+validateSignaturePayload :: SignatureAuthData -> (Either ErrorResponse TriggerSignatureOTPResp) -> FlowBT String Boolean
+validateSignaturePayload signatureAuth resp = 
+    case resp of
+        Right (TriggerSignatureOTPResp triggerSignatureOtpResp) -> do
+            case triggerSignatureOtpResp.person of
+                Just (User person) -> do
+                    lift $ lift $ setLogField "customer_id" $ encode (person.id)
```

Comment:
write a helper called `lift2` rather than calling `lift $ lift $` everyone. or another way is to write `setLogField` on `FlowBT` monad and use that.

``` purescript
 setLogField = lift $ lift $ X.setLogField
```